src/pages/home-page/hero-image/hero-image.module.scss

```diff
@@ -3,6 +3,9 @@
 .root {
     position: relative;
     color: $white;
+    width: 100vw;
```

Comment:
on purpose?

Reply:
what Mika and I said in the morning, we will keep both  - 
canvas margins right& left 0 + 100vw